bin/post_compile

```diff
@@ -80,21 +97,23 @@ heroku-install-cabot-plugins() {
   done
 }
 
+heroku-put-scripts-on-path() {
+  # A better way of doing this perhaps?
+  ln -sf /app/.heroku/vendor/node-4.2.6/bin/* /app/.heroku/python/bin/
+}
+
 heroku-compress-assets() {
   puts-step "Compressing assets"
-  PATH=$PATH:/app/$NODE_PATH/bin .heroku/python/bin/python manage.py compress | indent
+  PATH=$PATH:/app/$NODE_PATH/bin .heroku/python/bin/python manage.py compress --force | indent
 }
 
 
 main() {
-  if [[ ! -n $DYNO ]]; then
```

Comment:
The cabot people want this stuff to only run on heroku - though honestly `post_compile` is a buildpack-specific thing anyhow - hence the question.